content/article/the-rock-pi-s-or-how-to-get-gpio-input-without-a-library.md

```diff
@@ -0,0 +1,198 @@
+
+  {
+    "title"       : "The Rock Pi S; or, How to Get GPIO Input Without a Library",
+    "authors"     : ["timm-murray"],
+    "date"        : "2020-01-20T17:50:11",
+    "tags"        : [ "hardware", "linux", "gpio", "pi", "rockpi" ],
+    "draft"       : true,
+    "image"       : "/images/the-rock-pi-s-or-how-to-get-gpio-input-without-a-library/rockpi.jpg",
+    "thumbnail"   : "",
+    "description" : "There are lots of single board computers out there, but not all of them have an simple library for getting input on a pin. Never fret, because most of them run Linux, and it's easy to access pins with just Linux.",
+    "categories"  : "hardware"
+  }
+
```

Comment:
This is looking really good. There are a few things that can get it better traction though.

I'd add a topic paragraph here. I'm not a Pi person and I don't know what GPIO is. A motivating case ("I was trying to do this, and need to do that") can draw people into the article. Imagine that the first two sentences are the only things people will see in previews on Twitter, and so on.

I see the technical details in this article, but what else are you doing? Making blinking lights, etc? A concrete, physical example would make this pop a bit more.